assets/js/admin.js

```diff
@@ -557,6 +557,14 @@ jQuery(function($) {
 
                 dismissNotice( $notice, dismissible_id, nonce );
             } );
+
+        $( document ).on( 'click', '.wpbdp-admin-silent-dismiss', function( e ) {
```

Comment:
Would it work to add the "notice-dismiss" class on the link in the message instead of adding extra js?

Reply:
Hey Steph, it wouldn't work mainly because the `notice-dismiss` checks for `parent` which is the direct parent of the button. The link sits within  a parent element `p` hence the new js function uses `closest` https://stackoverflow.com/questions/10499435/parent-vs-closest

Another issue is the class `notice-dismiss` is styled to work as a button. Assigning it to the link has the following styling effect
![Screenshot 2021-11-20 092112](https://user-images.githubusercontent.com/121492/142716798-8402e25a-ef3b-49f4-b4bf-cf325b726726.png)

Reply:
Gotcha. Thanks for checking. 

Maybe we could reduce the duplication here a bit although it already existed? Maybe you could look into it to see if these are both being used? It's possible one is for a version of WordPress we no longer support.

That would probably help to move some of the duplication into the dismissNotice function.

Reply:
Yes, agreed :) Will check the code for this

Reply:
Done some changes to combine the js into one. Added a callback parameter that is used to send to the next page after the background dismiss action is passed. This is mainly because the ajax action is not completed before the link opens the new page. It ensure the ajax action is completed before sending to the new page